plugin-taskbar/lxqttaskbutton.cpp

```diff
@@ -395,7 +421,8 @@ void LXQtTaskButton::mouseMoveEvent(QMouseEvent* event)
 
     // release mouse appropriately, by positioning the event outside
     // the button rectangle (otherwise, the button will be toggled)
-    QMouseEvent releasingEvent(QEvent::MouseButtonRelease, QPoint(-1,-1), Qt::LeftButton, Qt::NoButton, Qt::NoModifier);
+    //FIXME: constructor is deprecated, also seems a bit hacky code
```

Comment:
This comment isn't needed; your code is correct. Please remove the comment.

Reply:
Still the code is a bit hacky and could be reworked. Don't know yet how.

Reply:
This of a code reader who might encounter "//FIXME: constructor is deprecated"  and ask, "which deprecated c-tor? This isn't deprecated."  I mean it'll be confusing later.